test/new-e2e/tests/windows/common/user.go

```diff
@@ -63,3 +86,49 @@ func DotSlashNameToLogonName(host *components.RemoteHost, user string) (string,
 	user = strings.TrimPrefix(user, ".\\")
 	return MakeDownLevelLogonName(hostname, user), nil
 }
+
+// GetADGroupMembers returns the list of members of the given AD group
+func GetADGroupMembers(host *components.RemoteHost, group string) ([]Identity, error) {
+	cmd := fmt.Sprintf(`ConvertTo-JSON -InputObject @(Get-ADGroupMember -Identity "%s" | Foreach-Object {
+		@{
+			Name = $_.Name
+			SID = $_.SID.Value
+		}})`, group)
+	out, err := host.Execute(cmd)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
This is an idea for the future (not this PR), but it might be cool to have a way to log (at Debug level) the exact command that was run if there were an error. Specifically for commands that use the format specifiers / dynamic arguments, if trying to debug a failing command, it'd make it really easy to run the exact command manually as give by the debug log statement.

Reply:
yep I'd love to have something like `host.Log.Debug()`